OpenRA.Game/Settings.cs

```diff
@@ -20,23 +20,61 @@ namespace OpenRA
 {
 	public enum MouseScrollType { Disabled, Standard, Inverted }
 
+	public class LaunchSettings
```

Comment:
My idea was that keeping track of them in a central place outweighs the hackiness of it.